packages/example/components/chains/suiStandard/example.tsx

```
@@ -39,6 +39,155 @@ import '@mysten/dapp-kit/dist/index.css';
+      }
+
+      acc.get(coinType)?.push(coin);
+
+      return acc;
+    }, new Map<string, CoinStruct[]>());
+
+    return coinTypes;
+  }
+
+  useEffect(() => {
+    if (currentAccount && currentAccount?.address) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)